core/settings.py

```
@@ -0,0 +1,767 @@
+        environ=True, environ_prefix=None, environ_name="RECAPTCHA_PUBLIC_KEY"
+    )
+    CRISPY_ALLOWED_TEMPLATE_PACKS = (
+        "bootstrap",
+        "uni_form",
+        "bootstrap5",
+        "bootstrap4",
+    )
+    CRISPY_TEMPLATE_PACK = "bootstrap5"
+    #!SECTION END - Application definition
+
+    # SECTION Start - Media, Files and Static Assests Storage
```

Pylint (4.0.1)

[warning] SECTION Start - Media, Files and Static Assests Storage

(W0511)